Changelog.md

```diff
@@ -1,9 +1,15 @@
-### Development
+### 5.0.0
 [Full Changelog](https://github.com/rspec/rspec-rails/compare/v4.1.0...main)
 
+Enhancements:
+
 * Support new #file_fixture_path and new fixture test support code. (Jon Rowe, #2398)
 * Support for Rails 6.1. (Benoit Tigeot, Jon Rowe, Phil Pirozhkov, and more #2398)
```

Comment:
Should we add more details about this?

Reply:
Please do!

Reply:
@benoittgt and @pirj, I intend to merge this and release 5.0 tomorrow unless either of you have anything you want to add (Im happy to delay a bit longer if so). :)

Reply:
After my message I looked at the commit list and found commits pretty technical or deprecation fix. I am wondering how much it is valuable to be in the changelog. 🤔 

I have no blocker for a release. I can work for the a blog post on rspec.info next week with infos about this release.

Thanks for the work.